model/roles.go

```diff
@@ -1501,6 +1501,17 @@ func (r *Role) LookupJob(name string) *RoleJob {
 	return nil
 }
 
+// IsPrivileged tests if the role capabilities enable fully privileged
+// mode.
+func (r *Role) IsPrivileged() bool {
+	for _, cap := range r.Run.Capabilities {
```

Comment:
I think this needs to be always upper case (regardless of which capability)? I believe we have some input normalization going on when we read in the role manifest; maybe upper case all the capabilities there, instead of doing this here (potentially repeatedly)?

Reply:
In the next line it uses `ToUpper` to normalize for the comparison.
The input normalization you think of is likely the `getSecurityContext` in pod.go.
See https://github.com/SUSE/fissile/pull/351/files/7e2a6f97e5369d540dc4db413e5f69473260cc4e#diff-a563c88f5954ff16e94d3ec81b1de52cR454

`IsPrivileged` is a variant of that which only does the checking for ALL without saving the transformed capabilities as `gSC` does.

Oh. You are asking to move the normalization from `IsP` / `gSC` to the where the manifest is read.
Should be possible. Will try to find where that happens.
Likely in the various validations.

Reply:
Done with commit ef1155c4a44ae9f883dfa6dddfd16cfa09a18d91